From: coolsnowwolf <coolsnowwolf@gmail.com>
Subject: [PATCH] nss-clients: consume external compiler flags

Linux 6.18 no longer consumes EXTRA_CFLAGS in the top-level Kbuild.
Add the package-provided NSS and SSDK includes to module flags.

Signed-off-by: coolsnowwolf <coolsnowwolf@gmail.com>
---
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,7 @@
 # Makefile for the clients using the NSS driver
 
 ccflags-y := -I$(obj) -I$(obj)/..
+subdir-ccflags-y += $(EXTRA_CFLAGS)
 
 export BUILD_ID = \"Build Id: $(shell date +'%m/%d/%y, %H:%M:%S')\"
 ccflags-y += -DNSS_CLIENT_BUILD_ID="$(BUILD_ID)"
